requirements.txt

```diff
@@ -1,6 +1,5 @@
 kombu==3.0.30
 Jinja2==2.8
-mozdownload==1.20.2
```

Comment:
This was actually a bad idea given that our Python files are still using it. I will push a bustage fix to get this fixed. Sorry for that.